src/lava/lib/optimization/solvers/generic/hierarchical_processes.py

```diff
@@ -15,9 +16,30 @@ class ContinuousVariablesProcess(AbstractProcess):
     """Process which implementation holds the evolution of continuous
     variables on the solver of an optimization problem."""
 
-    def __init__(self, **kwargs):
-        super().__init__(**kwargs)
-        raise NotImplementedError
+    def __init__(
+        self,
+        shape: ty.Tuple[int, ...],
+        problem: OptimizationProblem,
+        backend,
+        hyperparameters: ty.Dict[str, ty.Union[int, npt.ArrayLike]] = None,
+        name: ty.Optional[str] = None,
+        log_config: ty.Optional[LogConfig] = None,
+    ) -> None:
+        super().__init__(
+            shape=shape,
+            name=name,
+            problem=problem,
+            log_config=log_config,
+        )
+
+        self.num_variables = np.prod(shape)
+        self.backend = backend

```

Comment:
Does the Process still need knwoledge of its backend? The idea of the Process was that it is backend agnostic.

Reply:
it does